libraries/apollo-debug-server/src/androidMain/resources/schema.graphqls

```diff
@@ -0,0 +1,32 @@
+type Query {
+    version: Int!
+    clients: [Client!]!
+    normalizedCache(id: ID!): NormalizedCache!
+}
+
+type Client {
+    id: ID!
+    displayName: String!
+    normalizedCacheInfos: [NormalizedCacheInfo!]!
+}
+
+type NormalizedCacheInfo {
+    id: ID!
+    displayName: String!
+    recordCount: Int!
+}
+
+type NormalizedCache {
+    id: ID!
+    displayName: String!
+    clientDisplayName: String!
+    records: [KeyedRecord!]!
+}
```

Comment:
Let's make this GraphQL-y and merge `NormalizedCache` and `NormalizedCacheInfo`. 
I think that can remove the need for duplicating `displayName` and remove `clientDisplayName` completelly?

Listing clients:
```graphql
{
  clients {
    displayName
    normalizedCache {
      displayName
    }
  }
}
```

Getting records:

```graphql
{
  client(id) {
    normalizedCache {
      records
    }
  }
}
```

Maybe `NormalizedCache` doesn't even needs its own id?